ai-data/generative-apis/how-to/query-text-models.mdx

```diff
@@ -0,0 +1,164 @@
+---
```

Comment:
Pretty redundant with quickstart?

Reply:
Yes and no, goes much deeper than quickstart